src/scan.l

```diff
@@ -984,34 +984,54 @@ int yywrap()
 
 		for ( unsigned int i = 0; i < params.size(); ++i )
 			{
-			char* param = copy_string(params[i].c_str());
-			char* eq = strchr(param, '=');
-			char* val = eq + 1;
+			std::string pi = params[i];
+			auto first_non_id_char = 0;
 
-			*eq = '\0';
+			for ( size_t j = 0; j < pi.size(); ++j )
```

Comment:
An iterator here against the string would be faster.

Reply:
Thanks for the suggestion.  See what you think of the new commit.